packages/react-app/src/hooks/useStaticJsonRPC.js

```diff
@@ -9,14 +9,17 @@ const createProvider = async url => {
   return p;
 };
 
-export default function useStaticJsonRPC(urlArray) {
+export default function useStaticJsonRPC(urlArray, localProvider = null) {
   const [provider, setProvider] = useState(null);
 
   const handleProviders = useCallback(async () => {
     try {
       const p = await Promise.race(urlArray.map(createProvider));
       const _p = await p;
 
+      // set localProviders internal polling interval
+      // _p.pollingInterval = 50000;
+
```

Comment:
Why is this commented? 

The default (inside ethers providers) is 4000. So if RPC_POLLING_TIME is bigger, we cant set it, if not, leave the default